cg/services/run_devices/pacbio/metrics_parser/utils.py

```diff
@@ -32,12 +35,39 @@ def _parse_report_to_model(report_file: Path, data_model: Type[BaseMetrics]) ->
     parsed_json: dict = ReadFile.read_file[FileFormat.JSON](report_file)
     if data_model == SmrtlinkDatasetsMetrics:
         return data_model.model_validate(parsed_json[0], from_attributes=True)
-    metrics: list[dict[str, Any]] = parsed_json.get("attributes")
-    data: dict = {report_field["id"]: report_field["value"] for report_field in metrics}
+    metrics: list[dict[str, Any]] = parsed_json.get(MetricsFileFields.ATTRIBUTES)
+    data: dict = {
+        report_field["id"]: report_field[MetricsFileFields.VALUE_ONE] for report_field in metrics
+    }
     return data_model.model_validate(data, from_attributes=True)
```

Comment:
Does the "from_attributes" do anything here? If not, I think it adds confusion.

Reply:
This is to populate the class with the aliases, am I wrong? @islean

Reply:
The aliases are used by default!  My understanding is that `from_attributes` is more for if you send another object with overlapping field names. So if you pass a database SQLAlchemy Base object to a Pydantic DTO, you can set `from_attributes = True` and use a `PydanticObject.model_validate(database_object, from_attributes=True)` immediately. See https://docs.pydantic.dev/latest/concepts/models/#arbitrary-class-instances.

Reply:
Ok, will remove, thanks!